src/chi-vue/src/components/data-table-filters/AdvancedFilters.tsx

```diff
@@ -0,0 +1,287 @@
+import { Component, Prop, Vue } from 'vue-property-decorator';
+import { copyArrayOfObjects, findComponent, uuid4 } from '@/utils/utils';
+import { DataTableFilter, DataTableFormElementFilters } from '@/constants/types';
+import {
+  ACCORDION_CLASSES,
+  BUTTON_CLASSES,
+  CHECKBOX_CLASSES,
+  FORM_CLASSES,
+  ICON_CLASS,
+  INPUT_CLASSES,
+  PORTAL_CLASS,
+  SELECT_CLASSES,
+  UTILITY_CLASSES,
+} from '@/constants/classes';
+import { compareFilters, getElementFilterData, updateFilterData } from '@/components/data-table-filters/FilterUtils';
+import { DATA_TABLE_EVENTS } from '@/constants/events';
+import DataTableFilters from '@/components/data-table-filters/DataTableFilters';
+
+Vue.config.ignoredElements = ['chi-popover'];
+// This element ignore is necessary as currently chi-popover is not a
+// Chi Vue component but a web component loaded from an external source
+
+// eslint-disable-next-line
+declare const chi: any;
+@Component
+export default class AdvancedFilters extends Vue {
+  @Prop() filtersData?: DataTableFilter[];
+
+  // eslint-disable-next-line
+  advancedFiltersData?: DataTableFilter[];
+  _advancedFilterUuid?: string;
+  _advancedFilterButtonId?: string;
+  _advancedFilterPopoverId?: string;
+  _advancedFilterAccordionId?: string;
+  // eslint-disable-next-line
+  _advancedFiltersAccordion?: any;
+
+  created() {
+    this.advancedFiltersData = copyArrayOfObjects(this.$props.filtersData);
+    this._advancedFilterUuid = uuid4();
+    this._advancedFilterButtonId = `button__${this._advancedFilterUuid}`;
+    this._advancedFilterPopoverId = `popover__${this._advancedFilterUuid}`;
+    this._advancedFilterAccordionId = `accordion__${this._advancedFilterUuid}`;
+  }
+
+  _changeFormElementFilter(ev: Event, elementType: DataTableFormElementFilters) {
+    if (ev.target) {
+      const newFilterData = getElementFilterData(ev, elementType);
+
+      if (newFilterData && this.advancedFiltersData) {
+        updateFilterData(this.advancedFiltersData, newFilterData);
+        this.$forceUpdate();
+      }
+    }
+  }
+
+  _calculateFilterId(filterName: string) {
+    return `${this._advancedFilterPopoverId}_${filterName}`;
+  }
+
+  _createSelectFilter(filter: DataTableFilter, index: number) {
+    const options = filter.options?.map(option => {
+      return (
+        <option value={option.value} selected={filter.value === option.value}>
+          {option.label}
+        </option>
+      );
+    });
+
+    return (
+      <div
+        id={this._calculateFilterId(filter.name)}
+        class={`${FORM_CLASSES.FORM_ITEM} ${index !== 0 && !filter.advanced ? '-ml--2' : ''} ${
+          // Todo
```

Comment:
Classname needs to be updated once the HTML blueprint is ready. Related to FSDE-306.